internal/dev_server/sdk/go_sdk_test.go

```diff
@@ -123,4 +124,43 @@ func TestSDKRoutesViaGoSDK(t *testing.T) {
 			assert.Equal(t, value.AsArbitraryValue(), flagUpdate.NewValue.AsArbitraryValue())
 		})
 	}
+
+	newUpdates := map[string]ldvalue.Value{
+		"boolFlag":   ldvalue.Bool(true),
+		"stringFlag": ldvalue.String("fool"),
+		"intFlag":    ldvalue.Int(567),
+		"doubleFlag": ldvalue.Float64(99.99),
+		"jsonFlag":   ldvalue.CopyArbitraryValue(map[string]any{"tortoise": "hare"}),
+	}
+
+	// This test is testing the "put" payload in a roundabout way by verifying each of the flags are in there.
+	t.Run("Sync sends full flag payload for project", func(t *testing.T) {
+		trackers := make(map[string]*<-chan interfaces.FlagValueChangeEvent, len(updates))
```

Comment:
Channels are already pointer types, so I don't think you need the pointer to the channel.